preboots/wrhs.js

```diff
@@ -9,5 +9,16 @@ const Warehouse = require('warehouse.ai-api-client');
  */
 module.exports = function (app, options, next) {
   app.wrhs = new Warehouse(app.config.get('wrhs'));
+
+  const webhooks = app.config.get().webhooks || {};
```

Comment:
Is this what we want? to drive a set of webhooks via configuration? Or do we want to drive this via an endpoint where you can register a webhook and stuff the hooks in a backing store (dynamodb)

Reply:
I discussed this with both @jcrugzz and @decompil3d already. Short answer: in the long term   we could probably store this in a database. In the short term, having a config file speed up things and it makes it easier so that we can focus on the core part.